test/ComposerRequireCheckerTest/FileLocator/LocateAllFilesByExtensionTest.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace ComposerRequireCheckerTest\FileLocator;
+
+use ArrayObject;
+use ComposerRequireChecker\FileLocator\LocateAllFilesByExtension;
+use org\bovigo\vfs\vfsStream;
+use org\bovigo\vfs\vfsStreamDirectory;
+use PHPUnit\Framework\TestCase;
+
+class LocateAllFilesByExtensionTest extends TestCase
```

Comment:
Please add `/** @covers \ComposerRequireChecker\FileLocator\LocateAllFilesByExtension */` to prevent coverage leaks